From: Bastian Germann <bastiangermann@fishpost.de>
Date: Thu, 28 May 2020 18:21:43 +0200
Subject: Use gcc compiler

Use explicit gcc to enable cross compiling.
crossprefix-cc will not be available generally on Debian.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 943e013..852307b 100644
--- a/Makefile
+++ b/Makefile
@@ -183,8 +183,8 @@ include $(srctree)/scripts/Kbuild.include
 # this looks a bit horrible, but 'VAR ?= VALUE' preserves builtin values
 # rather than only user-supplied values from env or command-line
 $(call set_if_default_or_unset,AS,$$(CROSS_COMPILE)as)
-$(call set_if_default_or_unset,LD,$$(CROSS_COMPILE)cc)
-$(call set_if_default_or_unset,CC,$$(CROSS_COMPILE)cc)
+$(call set_if_default_or_unset,LD,$$(CROSS_COMPILE)gcc)
+$(call set_if_default_or_unset,CC,$$(CROSS_COMPILE)gcc)
 $(call set_if_default_or_unset,CPP,$$(CC) -E)
 $(call set_if_default_or_unset,AR,$$(CROSS_COMPILE)ar)
 $(call set_if_default_or_unset,NM,$$(CROSS_COMPILE)nm)
